Frontend/src/components/institutionOverview/InstitutionCard/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+import SettingsIcon from '@mui/icons-material/Settings';
+import Button from '@mui/material/Button';
+import ListCard from 'components/general/ListCard';
+import InstitutionLeaveButton from 'components/institutionOverview/InstitutionLeaveButton';
+import styles from './institutionCard.module.css';
+
+import getUser from 'shared/services/mock/user';
+
+function InstitutionCard({ institution, onLeft }) {
+  const [user, setUser] = useState({});
+  useEffect(() => {
+    getUser()
+      .then(setUser);
+  }, [setUser]);
+
+  const {
+    name, profilePictureURL, adminIds,
+  } = institution;
+  return (
+    <ListCard
```

Comment:
At some point this should be wrapped with a `Link`, right?

Reply:
No, a `ListCard` has an `onClick` prop, which also doesn't mess up the styling like I think a `Link` would.